server/src/main/java/org/elasticsearch/cluster/routing/allocation/MoveDecision.java

```diff
@@ -248,28 +257,39 @@ public String getExplanation() {
             if (canMoveDecision == AllocationDecision.AWAITING_INFO) {
                 return Explanations.Rebalance.AWAITING_INFO;
             }
+            // TODO (ES-13482): clusterRebalanceDecision is set to the result of AllocationDecider#canRebalance, which does not return
+            // NOT_PREFERRED or THROTTLE. This switch statement, and how MoveDecision uses clusterRebalanceDecision, should be
+            // refactored.
```

Comment:
I think it still does? see `org/elasticsearch/cluster/routing/allocation/decider/ConcurrentRebalanceAllocationDecider.java:182`

Reply:
Looks like Simon [committed some changes](https://github.com/elastic/elasticsearch/commit/5ef1c356f2aa18fa85001ead6ca837eafe98e99c) concurrently with my PR.

But I think I was wrong, anyway: I didn't realize the complexities of canRebalance.

Removed 👍 Thanks!